lesson10/hw10.py

```diff
@@ -0,0 +1,37 @@
+# номер 88
+
+
+nums = [4, 10, 20, 2, 1]        # я не знаю, как правильно попросить пользователя ввести список, поэтому сделала свой
```

Comment:
How to implement user list input:
1.
user_list = []
while variable !='x':
    variable = int(input("enter a list num")
    list.append(variable)
2. user_string = int(input('Input list use comma')) #1, 2, 3 ,4 
user_list= list(user_string)